hack/fake-apiserver/README.md

```diff
@@ -0,0 +1,46 @@
+# CAPM3 Fake API server
+
+Fake API server is a tool running inside a kubernetes cluster,
+and "generates" in-memory api server endpoints on demand.
```

Comment:
```suggestion
and "generates" in-memory k8s apiserver endpoints on demand.
```
Maybe like this? Then it is even more clear that it fakes target cluster k8s apiservers?